### DISCUSSION TITLE
copy url image after upload  in component SpatieMediaLibraryFileUpload

### DISCUSSION THREAD

**Discussion by farhad-rohani** [filamentphp/filament#1287] [Ideas]
I want to use of URL Laravel-medialibrary collection 
but 
in RichEditor does not work for this  

if  I have URL I can use  it in the content RichEditor Manually

thank you  for your good work

**Comment by danharrin** [comment]
Media library is for storing files associated with an Eloquent model - not just random files inside your database columns :)

**Reply by farhad-rohani** [reply]
yes, I understand.
but isn't good work that store all content image  in same folder.
